README.md

```diff
@@ -0,0 +1,28 @@
+# Roboto
+
+## Roboto sans-serif fonts for Toit devices
```

Comment:
As discussed in person: the first paragraph is used as description. -> Don't make it a title.
Actually reeds nicely too.
```suggestion
Roboto sans-serif fonts for Toit devices.
```